EventQueue.h

```diff
@@ -63,13 +63,21 @@ class EventQueue {
      *              value will dispatch events forever
      *              (default: -1)
      */
-    void dispatch(int ms=-1);
+    void dispatch(int ms);
+    void dispatch() { dispatch(-1); }
+
+    /** Break out of a running event loop
+     *
+     *  Already pending events may finish executing, but the queue will not
+     *  continue to loop indefinitely.
+     */
+    void break_();
```

Comment:
Could you rename this member function and clarify  what is going on when invoked ?